src/upgrade.js

```diff
@@ -152,35 +153,34 @@ Upgrade.process = function (files, skipCount, callback) {
 					date: date,
 				};
 
-				console.log('  → '.white + String('[' + [date.getUTCFullYear(), date.getUTCMonth() + 1, date.getUTCDate()].join('/') + '] ').gray + String(scriptExport.name).reset + '...');
+				process.stdout.write('  → '.white + String('[' + [date.getUTCFullYear(), date.getUTCMonth() + 1, date.getUTCDate()].join('/') + '] ').gray + String(scriptExport.name).reset + '...');
 
 				// For backwards compatibility, cross-reference with schemaDate (if found). If a script's date is older, skip it
 				if ((!results.schemaDate && !results.schemaLogCount) || (scriptExport.timestamp <= results.schemaDate && semver.lt(version, '1.5.0'))) {
-					readline.clearLine(process.stdout, 0);
-					readline.cursorTo(process.stdout, 0);
-					readline.moveCursor(process.stdout, 0, -1);
-					console.log('  → '.white + String('[' + [date.getUTCFullYear(), date.getUTCMonth() + 1, date.getUTCDate()].join('/') + '] ').gray + String(scriptExport.name).reset + '... ' + 'skipped'.grey);
+					process.stdout.write(' skipped\n'.grey);
 					db.sortedSetAdd('schemaLog', Date.now(), path.basename(file, '.js'), next);
```

Comment:
there is no `next` anymore

Reply:
Wasn't planning on changing the underlying code, so this line was left alone :grimacing: